docs/release-notes/identify-task.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  API: For tensorboard task types, fix a bug that would allow users to view tensorboards,
+      even if they did not have permission to view the all workspaces it included.
```

Comment:
:orphan:

**Bug Fixes**

-  TensorBoard: For TensorBoard task types, fix a bug that would allow users to view TensorBoards,
      even if they did not have permission to view the all workspaces it included.

Reply:
@carolinaecalderon, the release note seems unclear, i attempted to rewrite it